WordPress/Classes/Extensions/Notifications/NotificationBlock+Interface.swift

```diff
@@ -212,29 +212,37 @@ extension NotificationBlock {
     // MARK: - Constants
     //
     fileprivate struct Constants {
-        static let subjectRangeStylesMap: [NotificationRange.Kind: [NSAttributedStringKey: Any]] = [
-            .User: Styles.subjectBoldStyle,
-            .Post: Styles.subjectItalicsStyle,
-            .Comment: Styles.subjectItalicsStyle,
-            .Blockquote: Styles.subjectQuotedStyle,
-            .Noticon: Styles.subjectNoticonStyle
-        ]
+        static var subjectRangeStylesMap: [NotificationRange.Kind: [NSAttributedStringKey: Any]] {
```

Comment:
This is just curiosity: there are a bunch of changes like this (from a constant to a read-only var). What is the benefit of doing that? It seems like declaring something as a constant makes the expectation that it can not be changed explicit: a constant is assigned a value when it is declared. However, a var, even if it is read-only, carries a different meaning and works slightly differently (iirc, it will be calculated every time it is accessed, although in this case it is static, so that might not be the case)

I'm just trying to understand the rationale 😊

Reply:
+1 Wondering the reason of the `static let` change over for a calculated property. I'm placing my bet on lowering memory usage!.

If that's the case, we might wanna preserve the `static let` for those styles used in the Notifications List, since they'll be created over and over after scrolling every single row.

Reply:
The reason is to respond to the `UIContentSizeCategoryDidChange` notification.

We want the app to update it's contents size dynamically when the user changes the system font size.

The constants will store the first Font value it gets for the rest of app life, and that won't change even if the user changes the system font size.

With the computed property, when the style cache is cleaned, this will compute the new Style with the new font that our user wants to use.

I'm open to suggestions of better ways! I also felt sad making that change 😆